### PR TITLE
Fix max count issue in bin sorter

### PR DESCRIPTION
**Pull request by jianhuang01**
For #1360

#### Change List
- iterate items to get max count

### CODE REVIEW

File: src/core/utils/bin-sorter.js [hunk 1]
```diff
@@ -79,7 +79,9 @@ export default class BinSorter {
    * @return {Number | Boolean} max count
    */
   getMaxCount() {
-    return Math.max(...this.sortedBins.map(b => b.counts));
+    let maxCount = 0;
```
Comment (ibgreen) [inline src/core/utils/bin-sorter.js]: Nit: Just call it `max` or `maxValue`? It does not seem like a "count"...

File: src/core/utils/bin-sorter.js [hunk 2]
```diff
@@ -79,7 +79,9 @@ export default class BinSorter {
...
+    this.sortedBins.map(x => (maxCount = maxCount > x ? maxCount : x));
```
Comment (ibgreen) [inline src/core/utils/bin-sorter.js]: `map` creates and returns a new array of the same size. Better use `for of` or  possibly `forEach`.

**Comment by jianhuang01** [comment]
This is the last one which sorts an array in our code base.

File: src/core/utils/bin-sorter.js (hunk 1 above)
Comment (jianhuang01) [reply src/core/utils/bin-sorter.js]: trying to match with the function name.

**Pull request merged**